.github/workflows/nightly_lock.yaml

```diff
@@ -0,0 +1,14 @@
+name: nightly_lock
+on:
+  workflow_dispatch:
```

Comment:
Maybe we should call it lock and make a dispatch redo the calculation, e.g., run the lock again without the cache.

Reply:
I think it is easy enough to just delete the cache if this becomes a problem